.vscode/tasks.json

```diff
@@ -11,14 +11,6 @@
                 "cwd": "${workspaceRoot}"
             },
             "problemMatcher": [],
-        },
-        {
-            "label": "load_dials_env",
-            "command": "module",
-            "args": [
-                "load dials/latest"
-            ],
-            "type": "shell"
```

Comment:
While I appreciate it is annoying that it pops up when trying to debug in dodal, this should be removed as it's own ticket- if there isn's a ticket already pelase make one. I'm sure MX have some insight and can make this less disruptive